src/components/Game.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game: React.FC = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [history, setHistory] = useState<string[][]>([initialBoard])
+  const [isXNext, setIsXNext] = useState<boolean>(true)
+  const status = 'Next Player: ' + (isXNext ? 'X' : 'O')
+
+  /**
+   * based on the range value, sets the game to history[n]
+   * @param n
+   */
+  function jumpTo(n: number) {
```

Comment:
I'd also be more explicit for the name of this function. You pass this function around and use it in places where you don't really know the context (the game).

```suggestion
  function jumpToGameHistory(n: number) {
```